packages/muelu/test/scaling/scaling-gpu.xml

```diff
@@ -0,0 +1,58 @@
+<ParameterList name="MueLu">
+
+  <!-- ===========  GENERAL ================ -->
+  <Parameter        name="verbosity"                            type="string"   value="high"/>
+  <Parameter        name="number of equations"                  type="int"      value="1"/>
+
+  <!-- "unsmoothed" is cheaper in setup, but it also gives a worse preconditioner. -->
+  <Parameter        name="multigrid algorithm"                  type="string"   value="sa"/>
+
+  <Parameter        name="transpose: use implicit"              type="bool"     value="true"/>
+
+  <!-- ===========  AGGREGATION  =========== -->
+  <Parameter        name="aggregation: type"                    type="string"   value="uncoupled"/>
+  <Parameter        name="aggregation: drop scheme"             type="string"   value="classical"/>
+  <Parameter        name="aggregation: coloring algorithm"      type="string"   value="mis2 aggregation"/>
+  <!-- If we drop entries, we need to do so in symmetric fashion for MIS2 to work. -->
+  <!-- <Parameter        name="aggregation: symmetrize graph after dropping" type="bool" value="true"/> -->
+  <!-- <Parameter        name="aggregation: drop tol"                type="double"   value="0.02"/> -->
+
+  <!-- ===========  SMOOTHING  =========== -->
+  <Parameter        name="smoother: type"                       type="string"   value="CHEBYSHEV"/>
+  <ParameterList    name="smoother: params">
+    <Parameter      name="chebyshev: degree"                    type="int"      value="2"/>>
+    <Parameter      name="chebyshev: ratio eigenvalue"          type="double"   value="7"/>
+    <Parameter      name="chebyshev: min eigenvalue"            type="double"   value="1.0"/>
+    <!-- We get a better estimate by increasing the number of iterations. But this also increase the setup cost. -->
+    <Parameter      name="chebyshev: eigenvalue max iterations" type="int"      value="10"/>
+    <!-- If we have a good estimate, we can reduce the boost factor towards 1.0 -->
+    <Parameter      name="chebyshev: boost factor"              type="double"   value="1.1"/>
+  </ParameterList>
+
+  <!-- ===========  COARSE SOLVE  =========== -->
+  <Parameter        name="coarse: max size"                     type="int"      value="1000"/>
+  <Parameter        name="coarse: type"                         type="string"   value="CHEBYSHEV"/>
+  <!-- <Parameter        name="coarse: type"                         type="string"   value="tacho"/> -->
+  <!-- <Parameter        name="coarse: type"                         type="string"   value="cusolver"/> -->
+  <ParameterList    name="coarse: params">
+    <Parameter      name="chebyshev: degree"                    type="int"      value="6"/>>
```

Comment:
We should probably revisit the question of Chebyshev algorithm here since you're going higher order here, but don't let that block this PR.